src/Umbraco.Cms.Integrations.Crm.Dynamics/Helpers/EmbedCodeExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Umbraco.Cms.Integrations.Crm.Dynamics.Helpers
+{
+    public static class EmbedCodeExtensions
```

Comment:
Probably should be `StringExtensions` (as they are extensions on `string) and then for clarity name the method `ParseDynamicsEmbdCodeAttributeBalue`?